pkg/instancegroups/instancegroups.go

```diff
@@ -148,76 +149,49 @@ func (r *RollingUpdateInstanceGroup) RollingUpdate(rollingUpdateData *RollingUpd
 		}
 	}
 
-	for _, u := range update {
-		instanceId := u.ID
+	settings := resolveSettings(cluster, r.CloudGroup.InstanceGroup, numInstances)
 
-		nodeName := ""
-		if u.Node != nil {
-			nodeName = u.Node.Name
-		}
+	concurrency := 0
+	maxConcurrency := settings.MaxUnavailable.IntValue()
 
-		if isBastion {
-			// We don't want to validate for bastions - they aren't part of the cluster
-		} else if rollingUpdateData.CloudOnly {
+	if maxConcurrency == 0 {
+		klog.Infof("Rolling updates for InstanceGroup %s are disabled", r.CloudGroup.InstanceGroup.Name)
+		return nil
```

Comment:
Should we return an error here (as we did not update the IG)?

Reply:
We want to continue to update the subsequent instancegroups. The admin specifically wanted to have kops skip this instancegroup, so it isn't an error.

Reply:
One issue here is the placement. Where it is currently it will validate and taint before returning. Should this be moved up before the initial validation?

Reply:
I may be misunderstanding, but I think you're saying that `maxUnavailable: 0` means "skip this IG for rolling updates".   I'm not sure I follow that implication, but I also don't know what `maxUnavailable: 0` would mean!

Is `maxUnavailable: 0` something you're actually using?  Because we could just treat it as a validation error until we have maxSurge support (I guess that maxUnavailable=0 and maxSurge=0 is a validation error on a Deployment?)

I'm not overly worry about the tainting, TBH, given that!